.gitignore

```diff
@@ -11,3 +11,56 @@ _notebooks/.ipynb_checkpoints
 .netlify
 .tweet-cache
 __pycache__
+_posts/2020-03-11-Mortality_Rate.md
```

Comment:
This is not a robust way of ignoring files, I would suggest not doing this again you might break the build system

Reply:
Sorry my mistake, I will watch out for that in the future.